src/logs.ts

```diff
@@ -43,7 +43,7 @@ function processFrames(
   });
 }
 
-function orderFields(fields: Array<Field<any, any[]>>): Array<Field<any, any[]>> {
+function orderFields(fields: Array<Field<any>>): Array<Field<any>> {
```

Comment:
What's this change for?

Reply:
I believe this is cleanup from `main` the error thrown is `Generic type 'Field<T>' requires between 0 and 1 type arguments.`

Reply:
The linter told me to :scream: